src/ugeneui/src/app_settings/user_apps_settings/UserApplicationsSettingsGUIController.h

```diff
@@ -82,6 +82,16 @@ class UserApplicationsSettingsPageWidget : public AppSettingsGUIPageWidget, publ
 
 private slots:
     void sl_transFileClicked();
+
+private:
+    /**
+     * @brief STYLE_KEYS_FIXED_REGISTER
+     * The bunch of "Appearence" with fixed register.
+     * The first value - the original name, prodused by QStyleFactory::keys(), the second value - new name with fixed register.
+     * Note, that you can't replace any characters, only register could be changed
+     * (see here, https://doc.qt.io/qt-5/qstylefactory.html#details, "keys are case insensitive")
+     */
+    static const QMap<QString, QString> STYLE_KEYS_FIXED_REGISTER;
```

Comment:
nit: 'register' is a bad name here because as a noun it is a synonym of 'collection'
for the Upper/Lower case letters is is better to use "case" instead of "register": something like FIXED_CASE_QSTYLE_KEY_MAP

Reply:
Done